templates/default/less/Services/Calendar/delos.less

```diff
@@ -319,6 +322,15 @@ div.calevent {
 	margin: 3px;
 }
 
+div.calfullcontent {
+	position:absolute;
```

Comment:
Using absolute positioning is not an option here, since it prevents such a div to be proberly wrapped (e.g. by some wrapper dropzone). Please look for an alternative. Checkout: https://stackoverflow.com/questions/6319500/how-to-make-the-wrapping-div-element-with-relative-position-match-child-elements?rq=1.

Reply:
See comment above